k8s/upgrades/0.8.2-0.9.0/jiva/jiva_upgrade_runtask.yaml

```diff
@@ -300,6 +284,9 @@ spec:
     {{- jsonpath .JsonResult "{.items[*].metadata.labels.openebs\\.io/version}" | trim | saveAs "listReplicaDeployment.version" .TaskResult | noop -}}
     {{- .TaskResult.listTargetDeployment.version | notFoundErr "unknown openebs version" | saveIf "listReplicaDeployment.notFoundErr" .TaskResult | noop -}}
 
+    {{- jsonpath .JsonResult "{.items[*].spec.replicas}" | trim | saveAs "listReplicaDeployment.replicaCount" .TaskResult | noop -}}
+    {{- .TaskResult.listReplicaDeployment.replicaCount | notFoundErr "unknown no.of replicas in replica deployment" | saveIf "listReplicaDeployment.notFoundErr" .TaskResult | noop -}}
```

Comment:
it this really unknown no. of replicas or `No replicas found for jiva replica deployment`

Reply:
Updated the error message